test/Where.spec.ts

```diff
@@ -325,4 +325,70 @@ describe('Where', () => {
       expect(where).toEqual("owner.id:\"^(firstName>='Bob' AND firstName<'Boc' AND lastName>='Jones' AND lastName<'Jonet')\"");
     });
   });
+
+  describe('with group queries', () => {
+    it('should create a valid lucene query for A AND((B OR C) AND (D OR E))', () => {
+      const where = Where.toSearchSyntax({
+        id: 103,
+        $group_1: {
+          or: {
+            firstName: 'test',
+            lastName: 'test'
+          }
+        },
+        $group_2: {
+          or: {
+            owner: 'me',
+            title: 'me'
+          }
+        }
+      });
+      expect(where).toEqual('id:103 AND ((firstName:\"test\" OR lastName:\"test\")) AND ((owner:\"me\" OR title:\"me\"))');
```

Comment:
When I simplify this down, it looks like this is just double encapsulating the or statements, which doesn't result in the same thing mentioned in the test title:

```
A AND ( ( B OR C ) ) AND ( ( D OR E ) )
```

Reply:
Updated the tests to match their expected value.